.pre-commit-config.yaml

```diff
@@ -0,0 +1 @@
+/nix/store/2bvs1hdl54w2x8wjbh2icvv6wqn69dzj-pre-commit-config.json
```

Comment:
Is this file needed?

Reply:
Uh yeah, apparently not. Good catch!
Dunno if this has changed since I originally ran `devenv init` (that was a while ago). Nowadays it puts this file in `.gitignore`. So I'll just rebuild from scratch to get things setup to the latest configuration.